.eslintrc.js

```diff
@@ -15,6 +15,7 @@ module.exports = {
     },
     sourceType: 'module',
   },
+  ignorePatterns: ['/build/**/**/*.js'],
```

Comment:
Is the entire glob here necessary? Can we just ignore the `build` folder?

Reply:
That would have been easier yes, lemme fix that